README.md

```diff
@@ -51,6 +51,34 @@ DID of the w3access service.
 
 URL of the w3access service.
 
+### `REPLICATOR_ACCESS_KEY_ID`
+
+Access key for S3 like cloud object storage to replicate content into.
+
+### `REPLICATOR_SECRET_ACCESS_KEY`
+
+Secret access key for S3 like cloud object storage to replicate content into.
+
+### `REPLICATOR_ENDPOINT`
+
+Endpoint for S3 like cloud object storage to replicate content into.
+
+### `REPLICATOR_CAR_BUCKET_NAME`
+
+Bucket name to replicate written CAR files.
+
+### `REPLICATOR_INDEX_BUCKET_NAME`
+
+Bucket name to replicate written .idx files.
+
+### `REPLICATOR_DUDEWHERE_BUCKET_NAME`
```

Comment:
documenting them helps me see what they should be! I think these names can be env vars that are not tied to the replicator!

```suggestion
### `CARPARK_BUCKET_NAME`

Bucket name to replicate written CAR files.

### `SATNAV_BUCKET_NAME`

Bucket name to replicate written .idx files.

### `DUDEWHERE_BUCKET_NAME`
```

Reply:
IMO we might have the need in other places to have these names. Another stack can interact with SATNAV, CAR_BUCKET within AWS and need name for that.

Thoughts? I think replicator makes it clear these are from REPLICATOR context and not internal AWS ones. This was also a recommendation from Alan in previous review

Reply:
I'm imagining we'd set env vars in our SST build called `R2_CARPARK_BUCKET_NAME` and `S3_CARPARK_BUCKET_NAME` etc.... and the in the replicator we'd bind them like:

```js
Function(stack, 'carpark_replicator  { environment: {
  TARGET_BUCKET_NAME: R2_CARPARK_BUCKET_NAME
}})

Function(stack, 'satnav_replicator  { environment: {
  TARGET_BUCKET_NAME: R2_SATNAV_BUCKET_NAME
}})
```

Reply:
I also think R2 is better naming, specially as replicator can/will have other things. This was decided earlier, but I will revert then

Reply:
I renamed out level to be `R2_*` as suggested